datajunction-server/datajunction_server/api/nodes.py

```diff
@@ -833,6 +834,21 @@ def list_node_dag(
     return list(set(cast(List[Node], dimension_nodes) + downstreams + upstreams))  # type: ignore
 
 
+@router.get(
+    "/nodes/{name}/dimensions/",
```

Comment:
Q unrelated to this PR... you mentioned that somewhere we have a problem with the slashes at the end of the request. Isn't that b/c we add them here?

Reply:
It is unfortunately far more complex 😢 https://github.com/tiangolo/fastapi/discussions/7298

I'll look into adding one of the options there to DJ